### PR TITLE
README updated

### PR DESCRIPTION
**Pull request by hey-amanthakur**
#### Brief description of what is fixed or changed
fixed few grammatical errors in README.md

#### Release Notes 

NO ENTRY

### CODE REVIEW

**Comment by anutosh491** [comment]
Add "NO ENTRY" between the RELEASE NOTES header for allowing further tests to be executed like shown below 


NO ENTRY

**Comment by jhonsnow456** [comment]
Added NO ENTRY in release note

File: README.md [hunk 1]
```diff
@@ -253,7 +253,7 @@ A BibTeX entry for LaTeX users is
...
- author = {Meurer, Aaron and Smith, Christopher P. and Paprocki, Mateusz and \v{C}ert\'{i}k, Ond\v{r}ej and Kirpichev, Sergey B. and Rocklin, Matthew and Kumar, Amit and Ivanov, Sergiu and Moore, Jason K. and Singh, Sartaj and Rathnayake, Thilina and Vig, Sean and Granger, Brian E. and Muller, Richard P. and Bonazzi, Francesco and Gupta, Harsh and Vats, Shivam and Johansson, Fredrik and Pedregosa, Fabian and Curry, Matthew J. and Terrel, Andy R. and Rou\v{c}ka, \v{S}t\v{e}p\'{a}n and Saboo, Ashutosh and Fernando, Isuru and Kulal, Sumith and Cimrman, Robert and Scopatz, Anthony},
+ author = {Meurer, Aaron and Smith, Christopher P. and Paprocki, Mateusz and \v{C}ert\'{i}k, Ond\v{r}ej and Kirpichev, Sergey B. and Rocklin, Matthew and Kumar, Amit and Ivanov, Sergiu and Moore, Jason K.  and Singh, Sartaj and Rathnayake, Thilina and Vig, Sean and Granger, Brian E. and Muller, Richard P. and Bonazzi, Francesco and Gupta, Harsh and Vats, Shivam and Johansson, Fredrik and Pedregosa, Fabian and Curry, Matthew J. and Terrel, Andy R. and Rou\v{c}ka, \v{S}t\v{e}p\'{a}n and Saboo, Ashutosh and Fernando, Isuru and Kulal, Sumith and Cimrman, Robert and Scopatz, Anthony},
```
Comment (ThePauliPrinciple) [inline README.md:256]: Why was a space added here?

File: README.md [hunk 2]
```diff
@@ -270,7 +270,7 @@ A BibTeX entry for LaTeX users is
 ```
 
 SymPy is BSD licensed, so you are free to use it whatever you like, be
-it academic, commercial, creating forks or derivatives, as long as you
+it academic, commercial, creating forks, or derivatives, as long as you
```
Comment (ThePauliPrinciple) [inline README.md:273]: What issue does this comma solve?

**Pull request closed without merging**